src/main.rs

```diff
@@ -10,7 +10,7 @@ cfg_if::cfg_if! {
         mod awsv4;
         use chrono::Utc;
         use crate::awsv4::hash;
-        use fastly::handle::dictionary::DictionaryHandle;
+        use fastly::handle::config_store::ConfigStoreHandle;
```

Comment:
The low-level "handle" API's aren't intended for general use, and it'd be nice to direct people towards the friendlier interfaces that `ConfigStore` presents: https://docs.rs/fastly/latest/fastly/struct.ConfigStore.html

```suggestion
        use fastly::ConfigStore;
```